dlpy/utils.py

```diff
@@ -1101,10 +1101,16 @@ def create_object_detection_table(conn, data_path, coord_type, output,
 
     det_img_table = random_name('DET_IMG')
 
-    caslib, path_after_caslib = caslibify(conn, data_path, task='load')
+    caslib = find_caslib(conn, data_path)
```

Comment:
I believe I changed this to make sure that if there is already a caslib use it. Why changing back?

Reply:
It is not changing back. Basically, I need a caslib needs to be created to the data_path folder. After rereading your caslibify function,  I found that set task='save’ solve anything. I will change.

Reply:
I tried more tests and found both task='load' or task='save' works. Keep this part unchanged.